experimenter/experimenter/nimbus-ui/src/lib/constants.ts

```diff
@@ -44,14 +44,11 @@ export const EXTERNAL_URLS = {
     "https://experimenter.info/deep-dives/jetstream/metrics",
   // EXP-866 TBD URL
   PREVIEW_LAUNCH_DOC: "https://mana.mozilla.org/wiki/display/FJT/Nimbus",
-  RISK_BRAND:
-    "https://mana.mozilla.org/wiki/display/FIREFOX/Pref-Flip+and+Add-On+Experiments#PrefFlipandAddOnExperiments-Doesthishavehighrisktothebrand?",
+  RISK_BRAND: "https://experimenter.info/comms-sign-off",
   RISK_MESSAGE:
     "https://mozilla-hub.atlassian.net/wiki/spaces/FIREFOX/pages/208308555/Message+Consult+Creation",
-  RISK_PARTNER:
-    "https://mana.mozilla.org/wiki/display/FIREFOX/Pref-Flip+and+Add-On+Experiments#PrefFlipandAddOnExperiments-Isthisstudypartnerrelated?riskPARTNER",
-  RISK_REVENUE:
-    "https://mana.mozilla.org/wiki/display/FIREFOX/Pref-Flip+and+Add-On+Experiments#PrefFlipandAddOnExperiments-riskREV",
+  RISK_PARTNER: "https://experimenter.info/legal-sign-off",
+  RISK_REVENUE: "https://experimenter.info/vp-sign-off",
```

Comment:
Oh good, I just noticed those links are all busted as I'm reimplementing the overview form.  Perfect I'll incorporate this.